### ISSUE TITLE
[FR] Make an error thrown in song is not available in your country

### ISSUE THREAD

**Issue by CeaserPlayz** [SushiBtw/discord-music-player#61]
**Is your feature request related to a problem? Please describe.**
I recently fixed up my playlist command with the new update, but it suddenly stopped since one of the songs had one of those "The uploader has not made this video available in your country." things which stopped the bot from playing music. Also, if you used the .stop method, the bot wouldn't leave the VC and you also can't skip the song.

**Describe the solution you'd like**
Have the player throw an error if it is unavailable or can't be played, so that way it can be handled properly. If they video is unavailable, the bot would not join the VC or skip the song that is unavailable.

**Additional Comments**
The link to the song that was not playing is this: http://youtube.com/watch?v=aZenmeRytEM

**Comment by SushiBtw** [comment]
Hey, I will look into it and skip those songs.
I can also experience this issue.

**Comment by SushiBtw** [comment]
Note, that I can also reproduce it with videos that are deleted by YouTube. Working on a fix.

**Comment by SushiBtw** [comment]
Hey, I resolved the issue! Please check [v6.4.1 Update](https://www.npmjs.com/package/discord-music-player/v/6.4.1).

I have added a new Queue Handler:
```js
client.player.getQueue(guildID).on('songError', (errMessage, song) => {
    if(errMessage === 'VideoUnavailable')
        message.channel.send(`Could not play **${song.name}** - The song was Unavailable, skipping...`);
    else message.channel.send(`Could not play ${song.name} - ${errMessage}.`);
});
```

*If the error happens, the song is being skipped & event is being emitted.*

Thanks for reporting the issue 👍
